tests/server/test_plugins.py

```diff
@@ -2,40 +2,146 @@
 # -*- coding: utf-8 -*-
 # Copyright (C) 2007-2015, GoodData(R) Corporation. All rights reserved
 
-from multiprocessing import Queue
-from tempfile import mkstemp
+import copy
+import datetime
+import pytest
+from smoker.server import exceptions as smoker_exceptions
+import smoker.server.plugins as server_plugins
 import time
-import unittest
 
-from smoker.server.plugins import PluginManager, Plugin, PluginWorker
 
+class TestPluginManager(object):
+    """Unit tests for the PluginManager class"""
 
-class TestPluginManager(unittest.TestCase):
-    """
-    Unit tests for the PluginManager class
-    """
-    def test_example_config(self):
-        config = {}
-        config['plugins'] = {
-            'Uptime': {
-                'Category': 'system',
-                'Enabled': True,
+    conf_plugins_to_load = {
+        'CpuLoad': {
+            'Category': 'monitoring',
+            'Enabled': True,
+            'Command': 'top',
+            'Template': 'SystemPlugin'},
+        'Uname': {
+            'Category': 'system',
+            'Interval': 5,
+            'Module': 'smoker.server.plugins.uname'},
+        'Hostname': {
+            'Category': 'system',
+            'Command': 'hostname',
+            'Action': 'GetFQDN'}
+    }
+    conf_plugins_with_enabled_is_false = {
+        'Memory': {
+            'Category': 'system',
+            'Enabled': False,
+            'Interval': 15,
+            'Command': 'echo "$(top -l 1 | awk \'/PhysMem/\';)"',
+            'Timeout': 5}
+    }
+    conf_plugins_with_template_is_false = {
+        'Uptime': {
+            'Category': 'system',
+            'Enabled': True,
+            'Interval': 5,
+            'Command': 'uptime',
+            'Timeout': 5,
+            'Template': 'InvalidTemplate'}
+    }
+    conf_plugins = dict(conf_plugins_to_load.items() +
+                        conf_plugins_with_enabled_is_false.items() +
+                        conf_plugins_with_template_is_false.items()
+                        )
```

Comment:
this would look prettier like
```python
    conf_plugins = dict(
        conf_plugins_to_load.items() +
        conf_plugins_with_enabled_is_false.items() +
        conf_plugins_with_template_is_false.items())
```